### PR TITLE
Added result page

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Screenshot (57)

### CODE REVIEW

File: Result-page/result.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.main-wrapper {
+    background-color: rgb(12, 12, 12);
+    color: #a3a3a3;
```
Comment (tushargoyal812) [inline Result-page/result.css]: use color variables

File: Result-page/result.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    padding: 0;
+    height: 4rem;
+    width: 30rem;
+    margin: 1rem auto;
+    border-radius: 1rem;
+    cursor: pointer;
+}
+
+.alert-options {
+    background-color: rgb(37, 34, 34);
```
Comment (tushargoyal812) [inline Result-page/result.css]: here also use color variables

**Review by tushargoyal812 — COMMENTED** [review]
only minor changes needed and good to merge

File: Result-page/result.css (hunk 2 above)
Comment (syed-shaida-hussain) [reply Result-page/result.css]: Ok will do it
Thank you

**Review by sahiba0915 — COMMENTED** [review]
everything looks fine, just use color variables where you have used direct color names

**Pull request merged**